rsts/requirements.txt

```diff
@@ -0,0 +1,105 @@
+#
```

Comment:
can you add a make target to generate this file?

Reply:
I did add it, what happened, - oops overrode it